boards/samr21-xpro/include/periph_conf.h

```diff
@@ -286,6 +286,7 @@ static const i2c_conf_t i2c_config[] = {
 #define RTT_IRQ_PRIO        10
 #define RTT_ISR             isr_rtc
 #define RTT_MAX_VALUE       (0xffffffff)
+#define RTT_MIN_OFFSET      (10U)
```

Comment:
Is this really board specific?
I think this might as well live in `periph_cpu.h`.